Rename cc4me-network import to kithkit-a2a-client in sdk-bridge.ts

## Summary

`daemon/src/extensions/comms/network/sdk-bridge.ts` line 68 still imports `cc4me-network` (the old package name). This should be `kithkit-a2a-client`.

## Impact

- Build fails with `TS2307: Cannot find module 'cc4me-network'` on fresh installs or instances that only have `kithkit-a2a-client` installed
- Skippy's daemon cannot rebuild to pick up new routes until this is fixed
- BMO's daemon works only because `package.json` has BOTH `cc4me-network` and `kithkit-a2a-client` as deps — a workaround, not a fix

## Files Affected

All references to `cc4me-network` in `daemon/src/extensions/comms/network/`:
- `sdk-bridge.ts` line 68: `const sdk = await import('cc4me-network')`  
- `sdk-bridge.ts` line 71: warning message references `cc4me-network`
- `sdk-bridge.ts` line 7: JSDoc comment
- `sdk-bridge.ts` line 64: code comment
- `sdk-types.ts`: JSDoc comment references `cc4me-network`
- `crypto.ts`: JSDoc comment references `cc4me-network`

## Fix

Replace all `cc4me-network` references with `kithkit-a2a-client`. Update `package.json` to remove the `cc4me-network` dep if present.

## Found By

Skippy (build failure during daemon rebuild)

--- a/daemon/src/extensions/comms/network/sdk-bridge.ts
+++ b/daemon/src/extensions/comms/network/sdk-bridge.ts
@@ -4,7 +4,7 @@
  * Initializes the A2ANetwork client, wires SDK events to session bridge,
  * and exposes the network client for agent-comms (P2P fallback).
  *
- * cc4me-network is loaded dynamically. If not installed, daemon degrades
+ * kithkit-a2a-client is loaded dynamically. If not installed, daemon degrades
  * gracefully to LAN-only mode.
  */
 
@@ -61,14 +61,14 @@ export async function initNetworkSDK(config: R2Config): Promise<boolean> {
   }
 
   try {
-    // Dynamic import — cc4me-network is optional
+    // Dynamic import — kithkit-a2a-client is optional
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     let CC4MeNetworkClass: any;
     try {
-      const sdk = await import('cc4me-network');
-      CC4MeNetworkClass = sdk.CC4MeNetwork;
+      const sdk = await import('kithkit-a2a-client');
+      CC4MeNetworkClass = sdk.A2ANetwork;
     } catch {
-      log.warn('cc4me-network package not installed — P2P messaging unavailable. Install with: npm install cc4me-network');
+      log.warn('kithkit-a2a-client package not installed — P2P messaging unavailable. Install with: npm install kithkit-a2a-client');
       return false;
     }
 

